shared/metadata_test.go

```diff
@@ -109,7 +109,7 @@ func TestConstructMetadataResponse_OneLink(t *testing.T) {
 		},
 	}
 
-	MetadataResults := constructMetadataResponse(productSpecs, metadataMap)
+	MetadataResults := constructMetadataResponse(productSpecs, false, metadataMap)
```

Comment:
I think we are losing some test coverages because in fact `includeTestLevel` is `true` most of the time, as most of the usage is from /api/metadata. Could you add test-level metadata to some of these tests?

Reply:
(Ack, need to think about this a bit)